apps/src/codebridge/codebridgeContext/utils.ts

```diff
@@ -6,7 +6,9 @@ import {
 } from '@codebridge/types';
 import {useMemo} from 'react';
 
-import codebridgeI18n from '@cdo/apps/codebridge/locale';
+// disabling locales and falling back on the hardwired default due to apparent circular dep
```

Comment:
Can you file a ticket and link it here?

Reply:
Filed as https://codedotorg.atlassian.net/browse/CT-829